OpenSim/Simulation/Manager/Manager.h

```diff
@@ -59,7 +59,14 @@ class OSIMSIMULATION_API Manager
     Model *_model;
 
     /** Integrator. */
-    SimTK::Integrator* _integ;
+    // This is the actual integrator that is used when integrate() is called.
+    // Its memory is managed elsewehere; either by the user or by the
+    // _defaultInteg smart pointer.
+    SimTK::ReferencePtr<SimTK::Integrator> _integ;
+
+    // The integrator that is used when using the model-only constructor.
+    // This is allocated only if necessary.
+    SimTK::ResetOnCopy<std::unique_ptr<SimTK::Integrator>> _defaultInteg;
 
```

Comment:
There are many things that make this class behave poorly if copied. For example, the `_controllerSet` member variable would also need to be updated based on the cloned model. We would need to write a custom copy constructor. My preference would be to just disallow copying the Manager--it almost certainly does not work anyway--delete the copy constructor, etc. 

I cannot use a ClonePtr for the `_defaultInteg` because it requires the new `System` to function properly.